x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -2764,14 +2764,7 @@
     "tagCloud.vis.schemas.segmentTitle": "标记",
     "tagCloud.vis.tagCloudDescription": "一组字词，可根据其重要性调整大小",
     "tagCloud.vis.tagCloudTitle": "标签云图",
-    "tagCloud.visParams.fontSizeLabel": "字体大小",
-    "tagCloud.visParams.maxFontSizeAriaLabel": "标记字体最大大小",
-    "tagCloud.visParams.maxPixelsAriaLabel": "像素",
-    "tagCloud.visParams.minFontSizeAriaLabel": "标记字体最小大小",
-    "tagCloud.visParams.minPixelsAriaLabel": "像素",
     "tagCloud.visParams.orientationsLabel": "方向",
```

Comment:
Any reason the orientations label was spared?

Reply:
The reason of removing other labels that they were not in sentence case (we try to follow [EUI writing style guide](https://elastic.github.io/eui/#/guidelines/writing)) or labels become unnecessary, or they were changed.
Here the `tagCloud.visParams.orientationsLabel` stayed the same as it was -> so there was not a reason to remove it